redact: error marshalling when debug is true

In the redact service if I make a request with `debug=true` it returns a score for a word which seems to be a float, which the SDK will try to marshall as an int and the request will fail.

Here is a reproducible code sample:
```go
package main

import (
	"context"
	"fmt"
	"os"

	"github.com/pangeacyber/pangea-go/pangea-sdk/pangea"
	"github.com/pangeacyber/pangea-go/pangea-sdk/service/redact"
)

func main() {
	token := os.Getenv("PANGEA_TOKEN")
	domain := os.Getenv("PANGEA_DOMAIN")
	if token == "" || domain == "" {
		panic("no token or domain")
	}

	cfg := &pangea.Config{
		Token:    token,
		Domain:   os.Getenv("PANGEA_DOMAIN"),
		Insecure: false,
	}

	client := redact.New(cfg)
	resp, err := client.Redact(context.Background(), &redact.TextInput{
		Text:  pangea.String("poop face shit damn 123456 123-4543 134-121-1234"),
		Debug: pangea.Bool(true),
	})
	if err != nil {
		panic(err)
	}

	fmt.Println(*resp.Result.RedactedText)
	for _, rr := range resp.Result.Report.RecognizerResults {
		fmt.Println(*rr.Text, *rr.Score)
	}
}
```

This code will error with
```
panic: pangea: POST https://redact.aws.us.pangea.cloud/v1/redact: request_id: prq_rijzlyblwiqsrtpwcc43gw4o5kj2uux5, request_time: 2023-04-26T22:42:30.031426Z, response_time: 2023-04-26T22:42:30.090950Z, status: Success, summary: Success. Redacted 1 item(s) from text: Error: json: cannot unmarshal number 1.0 into Go struct field RecognizerResult.report.recognizer_results.score of type int.

goroutine 1 [running]:
main.main()
        /Users/craig/zandbox/pangea/main.go:32 +0x2bc
exit status 2
```

Another point: Note that it is actually quite a bit of work to view the `RecognizerResults`. Is there a way to make this easier by perhaps removing the pointers to primitives for fields which must be there?

--- a/pangea-sdk/service/redact/api.go
+++ b/pangea-sdk/service/redact/api.go
@@ -106,7 +106,7 @@ type RecognizerResult struct {
 	FieldType *string `json:"field_type"`
 
 	// Score is always populated on a successful response.
-	Score *int `json:"score"`
+	Score *float64 `json:"score"`
 
 	// Text is always populated on a successful response.
 	Text *string `json:"text"`

--- a/pangea-sdk/service/redact/integration_test.go
+++ b/pangea-sdk/service/redact/integration_test.go
@@ -44,6 +44,35 @@ func Test_Integration_Redact(t *testing.T) {
 	assert.Equal(t, 1, out.Result.Count)
 }
 
+func Test_Integration_Redact_DebugTrue(t *testing.T) {
+	ctx, cancelFn := context.WithTimeout(context.Background(), 20*time.Second)
+	defer cancelFn()
+
+	cfg := redactIntegrationCfg(t)
+	client := redact.New(cfg)
+
+	redacted := "My Phone number is <PHONE_NUMBER>"
+
+	input := &redact.TextInput{
+		Text:  pangea.String("My Phone number is 415-867-5309"),
+		Debug: pangea.Bool(true),
+	}
+	out, err := client.Redact(ctx, input)
+	if err != nil {
+		t.Fatalf("expected no error got: %v", err)
+	}
+
+	assert.NoError(t, err)
+	assert.NotNil(t, out.Result)
+	assert.Equal(t, redacted, *out.Result.RedactedText)
+	assert.Equal(t, 1, out.Result.Count)
+	assert.NotNil(t, out.Result.Report.RecognizerResults)
+	assert.NotEmpty(t, out.Result.Report.RecognizerResults)
+	assert.NotNil(t, out.Result.Report.RecognizerResults[0].Score)
+	assert.NotNil(t, out.Result.Report.RecognizerResults[0].Text)
+
+}
+
 func Test_Integration_Redact_NoResult(t *testing.T) {
 	ctx, cancelFn := context.WithTimeout(context.Background(), 20*time.Second)
 	defer cancelFn()
@@ -90,6 +119,36 @@ func Test_Integration_Redact_Structured(t *testing.T) {
 	assert.NotNil(t, out.Result)
 	assert.Equal(t, redacted, out.Result.RedactedData)
 	assert.Equal(t, 1, out.Result.Count)
+}
+
+func Test_Integration_Redact_Structured_DebugTrue(t *testing.T) {
+	ctx, cancelFn := context.WithTimeout(context.Background(), 20*time.Second)
+	defer cancelFn()
+
+	cfg := redactIntegrationCfg(t)
+	cfg.Retry = true
+	client := redact.New(cfg)
+
+	data := map[string]any{"phone": "415-867-5309"}
+	redacted := map[string]any{"phone": "<PHONE_NUMBER>"}
+
+	input := &redact.StructuredInput{
+		Data:  data,
+		Debug: pangea.Bool(true),
+	}
+	out, err := client.RedactStructured(ctx, input)
+	if err != nil {
+		t.Fatalf("expected no error got: %v", err)
+	}
+
+	assert.NoError(t, err)
+	assert.NotNil(t, out.Result)
+	assert.Equal(t, redacted, out.Result.RedactedData)
+	assert.Equal(t, 1, out.Result.Count)
+	assert.NotNil(t, out.Result.Report.RecognizerResults)
+	assert.NotEmpty(t, out.Result.Report.RecognizerResults)
+	assert.NotNil(t, out.Result.Report.RecognizerResults[0].Score)
+	assert.NotNil(t, out.Result.Report.RecognizerResults[0].Text)
 }
 
 func Test_Integration_Redact_Structured_NoResult(t *testing.T) {

